src/lib/components/InputChip/InputChip.svelte

```diff
@@ -67,6 +67,27 @@
 		return { val: val, id: Math.random() };
 	});
 
+	function resetFormHandler() {
+		value = [];
+	}
+
+	onMount(() => {
+		const inputElement: HTMLInputElement | null = document.querySelector('.phantom-input') as HTMLInputElement | null;
+
+		// Verify input element AND external form are present
+		if (!inputElement || !inputElement.form) return;
+
+		const externalForm: HTMLFormElement = inputElement.form;
```

Comment:
Didn't know that you could get the parent form from an input. Nice!

Does select offer the same functionality? If so, we could make use of the internal select used for form submission instead of adding another hidden dom element.

Reply:
I will look into this, I totally overlooked the fact that it uses select to keep track of the values under the hood, I would be suprised if it wouldn't work.

Reply:
> Didn't know that you could get the parent form from an input. Nice!
> 
> Does select offer the same functionality? If so, we could make use of the internal select used for form submission instead of adding another hidden dom element.


Alright using the select works 👍 makes it a lot less janky with ghost inputs.

Reply:
Excellent! 👍